### ISSUE TITLE
install.py returns wrong path

### ISSUE THREAD

**Issue by Termuellinator** [DavidRisch/steamvr_utils#1]
for me, install.py returns `python3 /home/termy/projects/git/steamvr_utils/./scripts/steamvr_utils.py daemon  >>/tmp/steamvr_utils.out 2>&1; %command%;` where it should be `python3 /home/termy/projects/git/steamvr_utils/scripts/steamvr_utils.py daemon  >>/tmp/steamvr_utils.out 2>&1; %command%;` (notice the /./ "subfolder" before /scripts )

Edit: and i just noticed, the desktop files contain the wrong paths, too:

```
Exec=python3 /home/termy/projects/git/steamvr_utils/./scripts/steamvr_utils.py off
Icon=/home/termy/projects/git/steamvr_utils/./scripts/../images/icon_off.png
```
So /. is too much on Exec and /./scripts/.. is too much on Icon

I guess that might be because i followed the instructions in the readme and ran `./scripts/install.py ` from the basedir?

**Comment by DavidRisch** [comment]
Fixed in b5cb9d45.
Technically the paths were not wrong, `a/./b` and `a/b` should refer to the same file on a UNIX system.
But paths without `.`/`..` definitely look nicer.

**Comment by Termuellinator** [comment]
oh, didn't know this! :)
